lib/apiservers/engine/backends/volume.go

```diff
@@ -53,6 +53,11 @@ var validDriverOptsKeys = map[string]struct{}{
 	DriverArgImageKey:     {},
 }
 
+var supportedVolDrivers = map[string]struct{}{
+	"local":   struct{}{},
```

Comment:
@hickeng this is indeed the case. The docker CLI sends the "local" driver to the personality if none is specified. I included the "vsphere" portion when I wrote the code to demonstrate that we can take whatever argument we want from the `driver` that is supplied from the docker CLI. This allows us to have different behavior if we so chose, but would not line up at all with docker's use of drivers. +1 on the only returning one driver, and if we do honestly it should be local. If we do return vsphere then we should intercept local and always interpret it silently as vsphere as well since that is our native "local"ly supported method of storage.